plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -515,7 +516,7 @@ open class HtmlRenderer(
         }
 
     private fun FlowContent.buildLink(to: PageNode, from: PageNode) =
-        buildLink(locationProvider.resolve(to, from)) {
+        buildLink(locationProvider.resolve(to, from)!!) {
```

Comment:
Same here and for all following uses of a the bang. Maybe it would also be fine to just ?: return